gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -2411,4 +2433,9 @@ private boolean requesterShouldPay(String bucketName) {
   public AtomicLong getObjectStatistics(GoogleCloudStorageStatistics key) {
     return objectStatistics.get(key);
   }
+
+  @Override
+  public AtomicLong getHttpStatistics(GoogleCloudStorageStatistics key) {
+    return httpStatistics.get(key);
```

Comment:
Please add a null check here

Reply:
Updated

Reply:
httpStatistics can still be null, we dont want to throw the exception here

Reply:
Updated